packages/core/docs/integrate/integration-guide.md

```diff
@@ -173,29 +196,16 @@ Each API function always contains `context` as a first parameter. This is the pl
 ## 2. Creating composables
 
 Composables are a major part of the integration. That exactly the place where the business logic comes in. We always serve this package as integration along with the corresponding Nuxt module.
-
-### 2.1 Exposing integration plugin wrapper
-
-The first thing that you do, is exposing a Nuxt plugin wrapper for the creation and configuring your integration by Nuxt environment. Within your composables package you have to add these few lines:
-
-```ts
-// composables/src/index.js
-import { integrationPluginFactory } from '@vue-storefront/core';
-import { createApiClient } from '@vue-storefront/commercetools-api';
-
-export const integrationPlugin = integrationPluginFactory(createApiClient);
-```
-
-That will expose the integration plugin wrapper, so the Nuxt will be able to create your integration within the context or extend already existing one, using a plugin
-
 ### 2.2 Creating nuxt module
 
 Inside of the composables packages you have to create another directory, next to `src` called `nuxt`. In that directory we need to place our Nuxt module. The Nuxt module is taking care of anything you want during the integration to be launched: adding plugins, injecting into the build process, creating some aliases, and more. The basic implementation of that module will add just a plugin that will configure our application (using a wrapper that you have already exposed)
 
+The plugin should also handle the option `useMiddleware` which is responsible for loading certain api-client. When `useMiddleware` is set to true, we need to load proxy version of api-client and start redirecting the traffic. Otherwise, we use direct connection.
```

Comment:
if we're usnure about this feature for CT (if it isn't redundant) should we require this from others?